test/raven.utils.js

```diff
@@ -108,6 +108,23 @@ describe('raven.utils', function() {
       parseStack([], callback);
       parseStack([{lol: 1}], callback);
     });
+
+    it('should extract context from last stack line', function(done){
```

Comment:
Alright, I will think about forcing a windows-style frame.filename and will commit again once ready. But that'll be tomorrow as here in Europe it is beer time now.